instances/treasury-devdao.near/widget/pages/dashboard/Portfolio.jsx

```diff
@@ -128,13 +128,17 @@ const PortfolioCard = ({
   return (
     <div className="d-flex flex-column">
       <div className="border-bottom">
-        <div className="py-2 d-flex gap-2 align-items-center justify-content-between px-3">
+        <div
+          className={`py-2 d-flex gap-2 align-items-center justify-content-between px-3 ${
+            !price ? "text-secondary" : ""
+          }`}
+        >
           <div className="d-flex align-items-center gap-2">
             <img src={icon} height={30} width={30} />
             <div>
               <div className="h6 mb-0">{symbol}</div>
               <div className="text-sm text-grey">
-                ${Big(price ?? "0").toFixed(2)}
+                ${Big(price ?? "0").toFixed()}
```

Comment:
To stay consistent with the decimal places
```suggestion
                ${Big(price ?? "0").toFixed(2)}
```